src/hooks/useArticle.js

```diff
@@ -42,3 +47,24 @@ export const useArticle = (lang, title) => {
 
   return [article, loadArticle]
 }
+
+const fixTableCaption = (htmlString, i18n) => {
+  const moreInformationText = i18n.i18n('more-information')
+  const hiddenClassName = 'hidden-in-table'
+  const parser = new DOMParser()
+  const node = parser.parseFromString(htmlString, 'text/html')
+  const tableNodes = node.querySelectorAll('table.wikitable, table.multicol')
+  for (const tableNode of tableNodes) {
+    const thContent = Array.from(tableNode.querySelectorAll('th')).map(th => th.textContent).join(', ')
+    const normalizedThContent = thContent.replace(/\[\d+]/g, '')
+    if (tableNode.caption && tableNode.caption.textContent) {
+      tableNode.caption.innerHTML = `<b class='${hiddenClassName}'>${moreInformationText}:</b><p class='${hiddenClassName}'>${normalizedThContent}</p><span>${tableNode.caption.textContent}</span>`

```

Comment:
This building of the UI definitely should be in <Table>

Reply:
same purpose and this addition more-information is used in article page.